catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -92,6 +92,21 @@
         <bundle>mvn:ddf.catalog.core/catalog-core-attachment-impl/${project.version}</bundle>
     </feature>
 
+    <feature name="poi" version="${project.version}">
+        <feature dependency="true">apache-commons</feature>
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlresolver/1.2_5</bundle>
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.saxon/9.9.1-5_1</bundle>
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.xmlbeans/3.1.0_1</bundle>
+        <bundle>mvn:org.apache.commons/commons-math3/3.6.1</bundle>
+        <bundle>mvn:org.apache.commons/commons-compress/1.19</bundle>
+        <bundle>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.poi/4.1.1_1</bundle>
```

Comment:
Is there a reason why we aren't using the same POI version throughout DDF? https://github.com/codice/ddf/blob/master/pom.xml#L291

Reply:
or at least be using https://github.com/codice/ddf/pull/5722/files#diff-600376dffeb79835ede4a0b285078036R291

Reply:
We ended up making a poi bundle to avoid embedding the libs everywhere. We should probably make a ticket to replace all of the other usages to use this poi bundle.